ietf/templates/meeting/requests.html

```diff
@@ -68,8 +68,8 @@ <h2 class="mt-5" id="{% firstof area.grouper.acronym "other-groups" %}">
                                     <i class="bi bi-people-fill"></i> {{ session.attendees }}
                                 </div>
                             {% endif %}
-                            {% if session.group.is_bof %}
-                                <span class="badge bg-success float-end">BOF</span>
+                            {% if session.group.state.slug != "active" %}
+                                <br><span class="badge bg-success">{{ session.group.state.name }}</span>
```

Comment:
Slightly concerned about bofs and proposed being the same color, but we can start here and improve later.